README.md

```diff
@@ -1961,6 +1961,39 @@ Other Style Guides
     }
     ```
 
+  <a name="no-use-before-define"></a><a name="14.5"></a>
+  - [14.5](#no-use-before-define) Variables, classes and functions should be defined before they can be used.
```

Comment:
```suggestion
  - [14.5](#no-use-before-define) Variables, classes, and functions should be defined before they can be used.
```

Reply:
Ty alot but still getting more issues lots bnb and omega coins by havker

Reply:
Whose dis